tox.ini

```diff
@@ -19,7 +19,7 @@ deps =
     -r{toxinidir}/requirements/dev.txt
     -r{toxinidir}/requirements/docs.txt
 commands =
-    make doc
+    sphinx-build -E -T -W -b html -D language=en -j auto -q docs/source docs/build  # make not on Windows
```

Comment:
https://stackoverflow.com/questions/2532234/how-to-run-a-makefile-in-windows may help with this.

Reply:
That's a heavy hack, installing all sort of non-native bloat. Sorry but no thanks!